test/commands/agent/validate/authoring-bundle.test.ts

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright 2025, Salesforce, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { expect } from 'chai';
+import { SfError } from '@salesforce/core';
+import { type CompilationError } from '@salesforce/agents';
+import AgentValidateAuthoringBundle, {
+  type AgentValidateAuthoringBundleResult,
+} from '../../../../src/commands/agent/validate/authoring-bundle.js';
+import { throwAgentCompilationError } from '../../../../src/common.js';
+
+describe('Agent Validate Authoring Bundle', () => {
+  describe('command configuration', () => {
+    it('should have correct summary', () => {
+      expect(AgentValidateAuthoringBundle.summary).to.equal('Validate an Agent Authoring Bundle');
+    });
+
+    it('should require project', () => {
+      expect(AgentValidateAuthoringBundle.requiresProject).to.be.true;
+    });
+
+    it('should have correct flags', () => {
+      const flags = AgentValidateAuthoringBundle.flags;
+      expect(flags).to.have.property('target-org');
+      expect(flags).to.have.property('api-version');
+      expect(flags).to.have.property('api-name');
+    });
+  });
+
+  describe('flag validation', () => {
```

Comment:
these tests should be removed, the --api-version flag validation is already tested in sf-plugins-core